config/300-webhooksource.yaml

```diff
@@ -57,6 +57,9 @@ spec:
                   context in which an event happened. Must be expressed as a URI-reference. Please refer to the
                   CloudEvents specification for more details: https://github.com/cloudevents/spec/blob/v1.0.1/spec.md#source-1"
                 type: string
+              corsOrigin:
```

Comment:
I'm sorry, but the name should reflect the full meaning of the header. How about `corsAllowOrigin`?

Reply:
Thank you!